Kitodo/src/main/java/org/goobi/mq/WebServiceResult.java

```diff
@@ -15,14 +15,15 @@
 
 import javax.jms.MapMessage;
 
-import org.apache.log4j.Level;
-import org.apache.log4j.Logger;
+import org.apache.logging.log4j.Level;
+import org.apache.logging.log4j.LogManager;
+import org.apache.logging.log4j.Logger;
 import org.joda.time.DateTime;
 import org.joda.time.format.DateTimeFormatter;
 import org.joda.time.format.ISODateTimeFormat;
 
 public class WebServiceResult {
-    private static final Logger logger = Logger.getLogger(ActiveMQDirector.class);
+    private static final Logger logger = LogManager.getLogger(ActiveMQDirector.class);
```

Comment:
Can you please correct the used class name from `ActiveMQDirectory.class` to `WebServiceResult.class` as the class name is `WebServiceResult`?